### ISSUE TITLE
Critical issues that are not critical (from my point of view)

### ISSUE THREAD

**Issue by h122015** [MISP/MISP#1816]
### Work environment

| Questions                 | Answers
|---------------------------|--------------------
| Type of issue             | Bug
| OS version (server)       | Ubuntu 
| OS version (client)       | Ubuntu
| PHP version               | 5.6
| MISP version / git hash   | 2.4.59, 2b187d48fccb69577fb714fd3c783aacd7ad217d
| Browser                   | FF

### Expected behavior
Not to show any critical issue for settings/modules that I want to have disabled.

### Actual behavior
In the server settings is stated in bright red that there are some critical issues that I need to address. Let's take the example of the setting site_admin_debug which by default is set to false. If I have a look at the column "Error Message" it says that the value hasn't been set. So if I click on the value to explicitly set it to false, I get the same error message. If I set it to true, this setting still remains red but now I see another error message "Enabling debug is not recommended. Turn this on temporarily if you need to see a stack trace to debug an issue, but make sure this is not left on."

Also in the plugins setting's page I see some red settings and apparently the value hasn't been set. But if I explicitly set the value to false, the issue remains red. It would be nice that if explicitly define the value as false, I don't get any more of these red issues.

**Comment by iglocska** [comment]
Indeed, I still need to fix this, some of the empty ones should not cause
critical issues if not set where it doesn't make much sense (we have an
option to allow "false" to be an accepted setting so it's just a matter of
revising all the settings that should have this set).

**Comment by ghost** [comment]
I had the same issue. I updated to 2.4.60 (Ubuntu 16.04.1 LTS) today and with a lot of fields the issue is resolved but the "site_admin_debug" field stays critical no matter what i put.
![site_admin_debug_false](https://cloud.githubusercontent.com/assets/20771149/22147890/54493168-df0b-11e6-9125-b5d252515cbc.JPG)
![site_admin_debug_true](https://cloud.githubusercontent.com/assets/20771149/22147894/56312d3c-df0b-11e6-8270-426f6b9a96a0.JPG)

**Comment by Turecky** [comment]
Have same problem here on 2.4.61 (Ubuntu 14.04.5 LTS)
